### PR TITLE
Byo sg

### PR DESCRIPTION
**Pull request by enderm**

### CODE REVIEW

File: docs/CONFIG-VARS.md [hunk 1]
```diff
@@ -66,14 +66,16 @@ To allow access for other administrative client applications (for example `kubec
...
 
 | <div style="width:50px">Name</div> | <div style="width:150px">Description</div> | <div style="width:50px">Type</div> | <div style="width:75px">Default</div> | <div style="width:150px">Notes</div> |
 | :--- | :--- | :--- | :--- | :--- |
 | default_public_access_cidrs | IP Ranges allowed to access all created cloud resources | list of strings | | Use to to set a default for all Resources |
 | cluster_endpoint_public_access_cidrs | IP Ranges allowed to access the AKS cluster api | list of strings | | for client admin access to the cluster, e.g. with `kubectl` |
-| vm_public_access_cidrs | IP Ranges allowed to access the VMs | list of strings | | opens port 22 for SSH access to the jump and/or nfs VM |
-| postgres_access_cidrs | IP Ranges allowed to access the AWS PostgreSQL Server | list of strings |||
+| <a name="vm_public_access_cidrs">vm_public_access_cidrs</a> | IP Ranges allowed to access the VMs | list of strings | | opens port 22 for SSH access to the jump and/or nfs VM |
```
Comment (thpang) [inline docs/CONFIG-VARS.md:77]: Remove html formatting items from here and line 78, 103

File: docs/CONFIG-VARS.md [hunk 2]
```diff
@@ -295,8 +300,8 @@ Each server element, like `foo = {}`, can contain none, some, or all of the para
 | multi_az | Specifies if PostgreSQL instance is multi-AZ | bool | false | |
 | deletion_protection | Protect from accidental resource deletion | bool | false | |
 | ssl_enforcement_enabled | Enforce SSL on connections to PostgreSQL server instance | bool | true | |
-| parameters | additional parameters for PostgreSQL server | list(map(string)) | [] | More details can be found [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.PostgreSQL.CommonDBATasks.html#Appendix.PostgreSQL.CommonDBATasks.Parameters) |
```
Comment (thpang) [inline docs/CONFIG-VARS.md:298]: This code should be in place not sure how your branched missed these. Pull 298 and 299 back into the file.

File: docs/CONFIG-VARS.md [hunk 3]
```diff
@@ -305,7 +310,7 @@ database_servers = {
   default = {
     administrator_password       = "D0ntL00kTh1sWay"
   },
-  another_server = {
```
Comment (thpang) [inline docs/CONFIG-VARS.md:308]: This entry is correct.

File: docs/CONFIG-VARS.md [hunk 4]
```diff
@@ -317,7 +322,7 @@ database_servers = {
     server_version               = "12"
     server_port                  = "5432"
     ssl_enforcement_enabled      = true
-    parameters                   = [{ "apply_method": "immediate", "name": "foo" "value": "true" }, { "apply_method": "immediate", "name": "bar" "value": "false" }]
```
Comment (thpang) [inline docs/CONFIG-VARS.md:320]: again, another correct entry removed.

File: main.tf [hunk 5]
```diff
@@ -340,7 +302,7 @@ module "kubeconfig" {
   depends_on = [ module.eks ]
 }
 
-# Database Setup - https://registry.terraform.io/modules/terraform-aws-modules/rds/aws/3.3.0
```
Comment (thpang) [inline main.tf:343]: More postgres items that need to be merged back into this pr.

File: main.tf [hunk 6]
```diff
@@ -387,6 +349,7 @@ module "postgresql" {
 
   multi_az = each.value.multi_az
 
+  # TODO - Look at simplifying contact logic
```
Comment (thpang) [inline main.tf:352]: Remove this line.

**Review by thpang — COMMENTED** [review]
Added comments and actions.

File: docs/CONFIG-VARS.md (hunk 1 above)
Comment (enderm) [reply docs/CONFIG-VARS.md:77]: html is the only way I found to achieve linking to items in markdown tables. Is there another way?

Comment (thpang) [reply docs/CONFIG-VARS.md:77]: We typically don't have links in md tables. Just reference the section. If you'd like to update every table in all our docs for every provider to enable table item linking, then create a PR for table linking in each provider.

Comment (enderm) [reply docs/CONFIG-VARS.md:77]: note that these anchors are being linked to by docs/BYONetwork.md

**Review by thpang — CHANGES_REQUESTED** [review]
@enderm there are some data server operator postgres items that are not in your branch here. Not sure if you merged from main or not, bu these need to be resolved. Once they are in place, I'll complete the review.

**Comment by enderm** [comment]
abandoning due to merge issues

**Pull request closed without merging**